firstfile.py

```diff
@@ -0,0 +1,29 @@
+"""Writing Python Code."""
+
+# !1.Заменить символ “#” на символ “/” в строке 'www.my_site.com#about'.
+Sq = 'www.my_site.com#about'.replace('#', '/')
```

Comment:
please read this PEP - https://peps.python.org/pep-0008/#descriptive-naming-styles